src/openrct2/config/Config.cpp

```diff
@@ -101,7 +101,7 @@ namespace OpenRCT2::Config
     });
 
     static const auto Enum_DrawingEngine = ConfigEnum<DrawingEngine>({
-        ConfigEnumEntry<DrawingEngine>("SOFTWARE", DrawingEngine::SoftwareWithHardwareDisplay),
+        ConfigEnumEntry<DrawingEngine>("SOFTWARE_HW", DrawingEngine::SoftwareWithHardwareDisplay),
```

Comment:
Should have been `SOFTWARE_HWD` - that’s what it used to be and the whole reason this PR was created in the first place.